### DISCUSSION TITLE
my experience with the Sony NW-ZX507 Android/Walkman and some things i noticed about FLAC customization

### DISCUSSION THREAD

**Discussion by impotentfield** [enzo1982/freac#311] [General Discussion]
i just want to share my experience with the Sony zx507 Walkman. with freac i can customize my Tool albums with flac. when i try to listen for the right setup i can feel the difference for a little while and then the acoustics go out of balance more and more until something starts to boil up in the capacitors and it comes back on and off. the quality, when the capacitor gets a hold of the signal just right is really noticeable especially with the flac custom files. i have a sony pha-2 headphone amplifier and that helps the capacitors to age a little and get a hold of the signal. in the android help application for the walkman, it say the capacitors have to attain highest quality sound over listening for some amount of time. its been taking a while since i got the walkman in march 2021. i just wanted to share my experience where i notice how good the quality of the flac customization and the walkman can produce. the sony headphone amplifier can help your success with quality in the capacitor units from your player that age and achieve higher quality in a permanent sense where the capacitors do some work and get stable and able to produce continually good sound. i just wanted to tell you guys what i myself have been up to with freac converter.

**Comment by zordaz** [comment]
What do you mean by 'Flac customization'? Can you give concrete examples? As already stated in other topics Flac is a lossless compression format, usage of the different Flac parameters will not and cannot have any effect on the sound. 

A headphone amplifier indeed can make a difference however.

**Comment by impotentfield** [comment]
goto my blog page about flac experimentation (www.alinthemodyne.blogspot.com) with apod in the flac encoder setup inside fre:ac. youll find a ton of setups with functions set up for putting inside the "Window functions" field area in the flac encoder setup window. if i were you i would try to encode with fast sinc interpolater and 16 bit sample format for signal processing and encode first with 29000, 29, 15, 15, 15 parameters, then reencode without the gauss functions in the window setup, and use streamable subset, 12, 15, 8, 8 parameters 'with' the section that says without gauss. the first encoding use the first section in the blog for the rounds, and then use without gauss for second reencoding. for my player that setup works for best quality and acoustic tightness.

**Comment by zordaz** [comment]
From what I understand now you are indeed just changing and experimenting with FLAC parameters. As stated earlier by myself and by others on this forum, this will not and simply can't have any impact on the sound. FLAC is a lossless compression format.

By adjusting FLAC parameters you will alter the compression efficiency, which only results in a smaller or bigger file, but the audio will always be the same. FLAC doesn't do sound processing.  If you want to do some sound processing you really have to use other software.
